gradle.properties

```diff
@@ -6,16 +6,16 @@ pluginName=Sourcegraph
 pluginVersion=6.0-localbuild
 # See https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html
 # for insight into build numbers and IntelliJ Platform versions.
-pluginSinceBuild=221.1
```

Comment:
If I understand correctly, we will be able to move to the new intellj platform plugin and resolve the FailureLevel.COMPATIBILITY_PROBLEMS issues (remove this failure level), right?

https://plugins.jetbrains.com/docs/intellij/tools-intellij-platform-gradle-plugin.html#requirements

Reply:
Yep